src/AddTodoForm.jsx

```diff
@@ -1,18 +1,23 @@
+import {useState} from "react";
+
 const AddTodoForm = ({onAddTodo}) => {
+    const [todoTitle, setTodoTitle] = useState("")
 
+    const handleTitleChange = (e) => {
+        const value = e.target.value;
+        setTodoTitle(value);
+    }
     const handleAddTodo = (e) => {
         e.preventDefault();
-        const todoTitle = e.target.elements.title.value;
-        onAddTodo(todoTitle);
-        e.target.elements.title.value = ''
-
+        onAddTodo({title: todoTitle, id: Date.now()});
+        setTodoTitle(" ")
```

Comment:
We should be careful to set this to an empty string (`""`) instead of a space (`" "`)!